loadtesting/proxy/tests/indexer.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import random
+
+
+from loadtesting.proxy.common.events import execute_before
+
+from locust import User, tag, task
+from loadtesting.proxy.common.base import NeonProxyTasksSet
+
+LOG = logging.getLogger(__name__)
+
+
+@tag("send_neon")
+class NeonIterativeTasksSet(NeonProxyTasksSet):
```

Comment:
we already have send_neon scenario and iterative trx scenatio
why do you need to duplicate it?